iym/tests/test_iym_uploads.py

```diff
@@ -0,0 +1,161 @@
+from django.test import TestCase
+from django.contrib.auth.models import User
+from django.core.files import File
+from iym.models import IYMFileUpload
+from budgetportal.models.government import FinancialYear
+
+import os
+import iym.admin
+import mock
+
+USERNAME = "testuser"
+EMAIL = "testuser@domain.com"
+PASSWORD = "12345"
+
+
+class MockResponse:
+    def __init__(self, json_data, status_code, text):
+        self.json_data = json_data
+        self.status_code = status_code
+        self.text = text
+
+    def json(self):
+        return self.json_data
+
+    def raise_for_status(self):
+        return None
+
+
+def mocked_requests_get(*args, **kwargs):
+    if "https://openspending-dedicated.vulekamali.gov.za/user/authorize" in args[0]:
```

Comment:
watch out of we make the OpenSpending base URL based on the setting, we need the mocks to work from the setting value too so that they always match

Reply:
done